docs/citing.md

```diff
@@ -0,0 +1,30 @@
+Citing a Dandiset
+-----------------
+
+If you use a Dandiset in your research, please acknowledge the Dandiset by citing it, just as you would a publication,
+including the DOI.
+The DOI can be found in the Dandiset's landing page on the DANDI Archive website.
+An example formatted citation can also be found on the Dandiset's landing page at the "CITE AS" button. This citation
```

Comment:
Perhaps a screenshot of an example DLP with arrows to the DOI and CITE AS locations would be helpful?